frontend/scripts/constants.js

```diff
@@ -32,6 +32,7 @@ export const NUM_DECIMALS = {
   tons: 0
 };
 export const DEFAULT_PROFILE_PAGE_YEAR = 2015;
+export const DEFAULT_PROFILE_PAGE_CONTEXT_ID = 1;
```

Comment:
Well, this is a very optimistic assumption :) It seems this is only used right now on the /profiles page, to provide a parameter for the search. Do we have plans of getting rid of this once we incorporate a "context switcher" on this page? In such case I'd leave a "TODO: " or "FIXME: " or something here.

Reply:
yes, this will be replaced by a dynamic variable, most likely by the end of this week, if all goes well.